test/tests/ver-1g/comparison_ver-1g.py

```diff
@@ -50,25 +48,19 @@
 gs = gridspec.GridSpec(1, 1)
 ax = fig.add_subplot(gs[0])
 
-analytical_time = np.linspace(0.0, 40, 40)
+tmap_sol = pd.read_csv("./gold/ver-1g_same_conc_out.csv")
+tmap_time = tmap_sol['time']
+tmap_concAB = tmap_sol['conc_ab']
+ax.plot(tmap_time, tmap_concAB, label=r"TMAP8", c='tab:gray')
+
+analytical_time = np.linspace(0.0, 40, 100)
 concA_o = 2.43e-4
```

Comment:
I cannot comment on the documentation since it was not updated, so I'm doing it here: 
The documentation should better explain how this value was obtained. It needs to state what temperature was selected for the conversion from pressure to concentration (I also don't see any in the TMAP4/7 V&V reports).